api/v1beta1/openstackcontrolplane_webhook.go

```diff
@@ -121,59 +116,10 @@ func (r *OpenStackControlPlane) ValidateDelete() error {
 // Default implements webhook.Defaulter so a webhook will be registered for the type
 func (r *OpenStackControlPlane) Default() {
 	openstackephemeralheatlog.Info("default", "name", r.Name)
-
-	if r.Spec.OpenStackClientImageURL == "" {
-		r.Spec.OpenStackClientImageURL = openstackControlPlaneDefaults.OpenStackClientImageURL
-	}
-
 	//
 	// set OpenStackRelease if non provided
 	//
 	if r.Spec.OpenStackRelease == "" {
-
-		var ospVersion OSPVersion
-		var err error
-
-		ospVersion, err = GetOSPVersionFromImageURL(r.Spec.OpenStackClientImageURL)
-		if err != nil {
-			openstackephemeralheatlog.Info("default", "name", r.Name)
-			ospVersion = OSPVersion(openstackControlPlaneDefaults.OpenStackRelease)
-		}
-
-		r.Spec.OpenStackRelease = string(ospVersion)
-	}
-}
-
-// GetOSPVersionFromImageURL -
-func GetOSPVersionFromImageURL(imageURL string) (OSPVersion, error) {
```

Comment:
@stuggi does this still make sense now that we are using digests? Guess it might still be useful for train/wallaby but we can just set it explicitly in those case

Reply:
we still use the version tagged URL to set the default images [1] which then points to the latest one using digest, or did I get it wrong?

[1] https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/config/default/manager_default_images.yaml#L15

Reply:
We should be replacing the tags with digests when building the CSV e.g https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/scripts/build_and_push_images.sh#L66

Not sure where this is done for the release CSV, @dprince is it scripts/stage_downstream_rebase.sh?

Reply:
ah it's the github action - https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/.github/create_bundle.sh